README.md

```diff
@@ -181,8 +181,7 @@ To change it:
 let g:go_highlight_functions = 1
 let g:go_highlight_methods = 1
 let g:go_highlight_fields = 1
-let g:go_highlight_structs = 1
-let g:go_highlight_interfaces = 1
+let g:go_highlight_types = 1
```

Comment:
This is a breaking change. Is there a way to not break this? I don't want people change their `vimrc` for no reason if possible :)